### PR TITLE
refactor(markdown-docx): export getClass as helper - #397

### PR DESCRIPTION
**Pull request by K-Kumar-01**
Signed-off-by: k-kumar-01 <kushalkumargupta4@gmail.com>

Need: While working on `LIST_RULE`, there was requirement of `node.$class`. Since a function has already been created in transformer, the decision was to export it as a helper and use it.

### Changes

-  use getClass as helper instead of transformer class function

### Author Checklist
- [x] Ensure you provide a [DCO sign-off](https://github.com/probot/dco#how-it-works) for your commits using the `--signoff` option of git commit.
- [ ] Vital features and changes captured in unit and/or integration tests
- [x] Commits messages follow [AP format](https://github.com/accordproject/techdocs/blob/master/DEVELOPERS.md#commit-message-format)
- [ ] Extend the documentation, if necessary
- [ ] Merging to `master` from `fork:branchname`

### CODE REVIEW

File: packages/markdown-docx/src/CiceroMarkToOOXML/helpers.js [hunk 1]
```diff
@@ -96,4 +96,14 @@ function wrapAroundDefaultDocxTags(ooxml) {
...
-module.exports = { sanitizeHtmlChars, titleGenerator, wrapAroundDefaultDocxTags };
+/**
+ * Gets the class of a given CiceroMark node.
+ *
+ * @param {object} node CiceroMark node entity
+ * @returns {string} Class of given node
+ */
+function getClass(node) {
+    return node.$class;
+}
```
Comment (algomaster99) [inline packages/markdown-docx/src/CiceroMarkToOOXML/helpers.js:107]: I would advise against this because `getClass` is supposed to be _bound_ to the class as you will have access to "node" only in the class, so it doesn't make sense to expose it. Moreover, helper methods don't have such strict parameter requirement. For example, `sanitizeHtmlChars` is a perfect example of a helper method because it can be a static method too as it just needs a string and doesn't matter from where we invocate it. However, `getClass` depends strongly on the input because of the `$class` of `node` and you will only get this while iterating over CiceroMark. So it's better to keep this as an instance method only.

To summarise, helper methods are those functions that are not much dependent upon the state of the application and can be reused throughout the code. You can also think of them as static methods in a class.

**Review by algomaster99 — COMMENTED** [review]
Although a nice initiative, I disagree with this design. See comment.

File: packages/markdown-docx/src/CiceroMarkToOOXML/helpers.js (hunk 1 above)
Comment (K-Kumar-01) [reply packages/markdown-docx/src/CiceroMarkToOOXML/helpers.js:107]: @algomaster99 Shall I close this PR then?

Comment (algomaster99) [reply packages/markdown-docx/src/CiceroMarkToOOXML/helpers.js:107]: If you are with my stand, then yes.

Comment (K-Kumar-01) [reply packages/markdown-docx/src/CiceroMarkToOOXML/helpers.js:107]: @algomaster99 Yeah, I think it makes sense but I have one doubt. So we want `node.$class` in the list transformer.
Should any function returns a node.$class or writes it as `node.$class` in the comparison itself?
We currently use `this.getClass` in the transformer itself and the rule will be imported from another file.

Comment (algomaster99) [reply packages/markdown-docx/src/CiceroMarkToOOXML/helpers.js:107]: Try to decouple rules from the node object. I feel there should be a way.
